src/devices/Hmc5883l/SamplesAmount.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Hmc5883l
+{
+    /// <summary>
+    /// Number of samples averaged (1 to 8) per measurement output.
+    /// </summary>
+    public enum  SamplesAmount: byte
+    {
+        /// <summary>
+        /// 1 (Default) samples per measurement output.
+        /// </summary>
+        One = 0b00000000,
```

Comment:
NIT: We usually try to use underscore to separate every 4 bits like `0b_0000_0000`. Makes it more readable. Same applies for the rest of the values.

Reply:
Fixed